src/BNO055.cpp

```diff
@@ -13,80 +13,69 @@ IMU::BNO055::BNO055(IO::I2C& i2C, uint8_t i2cSlaveAddress) : i2c(i2C) {
  * @return a boolean indicating whether or not the device was successfully initialized
  */
 bool IMU::BNO055::setup() {
```

Comment:
Instead of returning a boolean here, you it would be a little cleaner if you defined an enum specific to the BNO that identifies what kind of error occurred